public/javascripts/cohortController.js

```diff
@@ -14,3 +14,12 @@ var cohortController = function($scope,  $http, $state) {
     });
   };
 };
+
+var cohortStudentsController = function($scope, $http, $state) {
+
+  $scope.showStudent = function(student) {
+    $scope.$parent.$parent.currentStudent = student;
```

Comment:
that's what i'd assume, but in the view student data controller I'm just using `$scope.currentStudent` and it works. when i did the parent thing, the controller wouldn't change your current student to be the newly clicked one